cli/main.rs

```diff
@@ -701,6 +701,10 @@ async fn run_command(
     return run_from_stdin(flags).await;
   }
 
+  if !flags.has_permission() && flags.has_permission_in_argv() {
+    log::warn!("Permission flags have likely been incorrectly set after the module name. To grant permissions, define them before the module name.")
+  }
+
```

Comment:
Looks good! Please add two integration tests to `run_tests.rs` - one that uses `--quiet` flag and one the doesn't (the former shouldn't print this warning), where you put permission flags after the script name. After that I think it will be good to land 👍